app/models/authenticator/httpd.rb

```diff
@@ -4,6 +4,10 @@ def self.proper_name
       'External httpd'
     end
 
+    def self.group_delimiters
+      ::Settings.authentication.http_group_delimiters
```

Comment:
http doesn't make sense, but maybe `httpd` does or perhaps just remove the prefix altogether.

```suggestion
      ::Settings.authentication.httpd_group_delimiters
```

or

```suggestion
      ::Settings.authentication.group_delimiters
```

Reply:
haha, maybe `external_group_delimiters`... naming is hard